executor-utils/Cargo.toml

```diff
@@ -8,5 +8,6 @@ repository.workspace = true
 
 [dependencies]
 powdr-number.workspace = true
+powdr-ast.workspace = true
```

Comment:
@leonardoalt Does this sabotage #2124?

Reply:
not exactly because plonky3 already depends on ast as well, but we'll probably want to change that at some point too